app/src/main/java/com/kamron/pogoiv/scanlogic/Data.java

```diff
@@ -25,6 +25,9 @@
             0.767397165298462f, 0.770297293677362f, 0.773186504840851f, 0.776064947064992f, 0.778932750225067f,
             0.781790050767666f, 0.784636974334717f, 0.787473608513275f, 0.790300011634827f};
 
+    public static final int[] powerUpCandyCost = {1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1, 2,2,2,2,2,2,2,2,2,2,2,2,2,
```

Comment:
Between the 1s and the 2s looks to be a whitespace that shouldn't be there ;)

Reply:
👍